backend/app/services/session.py

```diff
@@ -46,3 +48,23 @@ async def delete_session(db_session: AsyncSession, session_id: str, current_user
 
     await db_session.delete(session_obj)
     await db_session.commit()
+
+async def get_session_ranking(db_session: AsyncSession, session_id: str) -> list[UserSessionRankingOut]:
+    session = await get_session_by_id(db_session, session_id)
+    participants = session.participants
```

Comment:
Does it work? Lazy loading doesn't work with async sqlalchemy. To access such attributes you have to await them 
```suggestion
    participants = await session.awaitable_attrs.participants
```